index.html

```diff
@@ -24,10 +24,17 @@
         <div class="col px-lg-5">
           <div class="jumbotron">
             <h1 class="display-4">Your local temperature</h1>
-            <hr class="my-4" />
             <span>The temperature is</span>
             <span id="theTemp">99</span>
             <span>° Fahrenheit.</span>
+            <hr class="my-4" />
+            
+            <!-- The following displays the current (static) location. -->
+            
+            <h1 class="display-5">Current Location</h1>
+            <span id="currentLocation">Nashville, TN</span>
```

Comment:
CSS naming convention is normally hyphen-case. Would you mind changing this to `current-location`?

If you do, I'll change `theTemp` above 😁